src/Extension.hx

```diff
@@ -12,14 +11,17 @@ class Extension {
 			?token:CancellationToken):ProviderResult<DebugConfiguration> {
 		var config:DebugConfiguration & Arguments = cast config;
 
-		if (Sys.systemName() == "Mac" && !Fs.existsSync('/usr/local/lib/libhldebug.dylib')) {
-			final visitButton = "Get from GitHub";
-			Vscode.window.showErrorMessage("Your version of Hashlink does not support debugging on Mac. Install a newer version from here:", visitButton).then(function(choice) {
-				if (choice == visitButton) {
-					Vscode.env.openExternal(Uri.parse("https://github.com/HaxeFoundation/hashlink"));
-				}
-			});
-			return null;
+		if (Sys.systemName() == "Mac") {
+			var hlVersion:String = js.node.ChildProcess.execSync('hl --version');
```

Comment:
This is not safe. It's possible to specify a custom path to the HL binary in the launch config.

https://github.com/vshaxe/hashlink-debugger/blob/master/src/Arguments.hx#L6

Reply:
I see. I‘ll change it so it takes the version specified in the config. The `libhl` path isn‘t configurable anywhere though and both hl and libhl need to be up-to-date in order for the debugger to work. Any suggestions? Should we assume libhl resides in the same directory as hl if a custom path is specified and if not assume it‘s in the default installation path (`/usr/local/lib`)?

Reply:
I pushed changes that use the `hl` specified in the launch configuration (if set). I also based the `libhl.dylib` location on that custom path (same folder), as we should assume that people pull a new Hashlink version from Github and use that version for testing without installing it system-wide. If no custom hl exec is set, then `libhl` is used from its default install location (`/usr/local/lib`).
I also added a check if the executable is properly codesigned, as I would expect it to be a common source of errors and it's hard to detect.